app/views/components/docs/summary.yml

```diff
@@ -0,0 +1,39 @@
+name: Summary
+description: Let users check their input before proceeding forward
+body: |
+  This component was build based on [Check answers pattern](https://design-system.service.gov.uk/patterns/check-answers/)
+part_of_admin_layout: true
+shared_accessibility_criteria:
+  - link
+accessibility_criteria: |
+  The change/edit link must:
+
+  * indicate which item or section reffers to
```

Comment:
Updated with an example. The accessibility criteria should not get into implementation details, that's why I had to keep it as a generic description of the behaviour. It refers to the fact that the link text should indicate which item or section refers to (e.g. instead of "Change" it should be "Change title" for screen-readers) - implemented this looks as shown below:
```
Change<span class="govuk-visually-hidden"> title</span>
```